src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final ColorSupplier supplierOfColors = new ColorSupplier();

```

Comment:
```suggestion
    private final ColorSupplier colorSupplier = new ColorSupplier();
```
such a name will suffice